pages/common/logstash.md

```diff
@@ -0,0 +1,12 @@
+# logstash
+
+> Migrate data from one source into another.
+> Usually used to ingest data from various other data sources into elasticsearch.
+
+- Check validity of a logstash configuration:
```

Comment:
I was going through logstash documentation. I think adding the `-e` example at first will help out beginners. Because the structure and syntax of the config file is not mentioned, and putting up something like - 
```
logstash -e 'input { stdin { } } output { stdout {} }'
```
will ease the users slowly into getting started with logstash.